sass/components/_videos.scss

```diff
@@ -102,6 +102,6 @@
     }
 
     .video-loading {
-        height: 360px;
+        height: 373px;
```

Comment:
why is this changing?

Reply:
This is an intended change as we are not using the right dimensions for placeholder this causes a scroll-pop with youtube videos. 

Screenshots are with same code but just the difference of heights

Before:
![nov-08-2018 14-35-02](https://user-images.githubusercontent.com/4973621/48188395-b43d8c00-e363-11e8-982c-0ef357785f27.gif)

After:
![nov-08-2018 14-37-59](https://user-images.githubusercontent.com/4973621/48188539-01216280-e364-11e8-8e6a-f70e5a425a88.gif)